plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/shipping-package-item-icon/index.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import type { CartItem } from '@woocommerce/types';
+
+/**
+ * Internal dependencies
+ */
+import type { PackageItem } from '../shipping-rates-control-package/types';
+import ProductImage from '../product-image';
+
+interface ProductImageProps {
+	packageItem: PackageItem;
+	cartItems: CartItem[];
+}
+/**
+ * Formats and returns an image element.
+ *
+ * @param {Object} props       Incoming props for the component.
+ * @param {Object} props.image Image properties.
+ */
+
+const ShippingPackageItemIcon = ( {
+	packageItem,
+	cartItems = [],
+}: ProductImageProps ): JSX.Element => {
+	const cartItem = cartItems?.find(
+		( item ) => item.key === packageItem.key
+	);
+	const images = cartItem?.images || [];
+
+	return (
+		<>
+			<ProductImage
+				image={ images.length ? images[ 0 ] : {} }
+				fallbackAlt={ cartItem?.name || '' }
+			/>
+		</>
```

Comment:
Does this have to be in a fragment, can't we just return the product image as is?